icepyx/core/APIformatting.py

```diff
@@ -242,15 +300,7 @@ def poss_keys(self):
 
     #     return self._wanted
 
-    @property
-    def fmted_keys(self):
-        """
-        Returns the dictionary of formatted keys associated with the
-        parameter object.
-        """
-        return self._fmted_keys
-
```

Comment:
It's not causing any failures because it isn't actually removed :) I moved the logic into a "descriptor":

https://github.com/icesat2py/icepyx/blob/434c28a64dcd7648684f4b30b170fde67146b843/icepyx/core/APIformatting.py#L198-L260

I did this because the `@overload` (which I used to represent the different return type possibilities based on `self.partype`) and `@property` decorators don't work together as expected.

Reply:
Okay... I hadn't connected all the pieces of logic to trace out where the descriptor returns the instance. Thanks!